.github/workflows/commands.yml

```diff
@@ -66,10 +69,21 @@ jobs:
           ref: ${{ steps.comment-branch.outputs.head_ref }}
 
       - name: Download extension
+        id: download-extension
         env:
           BODY: ${{ github.event.comment.body }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          curl -L $(echo $BODY | grep -ioEh 'https?://\S+.zip' -m1) -o /tmp/ext.zip
+          FILEURL=$(echo $BODY | grep -ioEh 'https?://\S+.zip' -m1)
+          FILENAME=$(basename $FILEURL)
+
+          if [[ "${FILENAME,,}" == *"example"* ]]; then
+            echo "example=true" >> $GITHUB_OUTPUT
+            gh pr comment ${{ github.event.issue.number }} --body "✅ Cannot update the example via command. Please feel free to upload the updated example zip file in a comment!"
+            exit 0
+          fi
+
+          curl -L $FILEURL -o /tmp/ext.zip
```

Comment:
This check seems a bit fragile, it could still allow for an example to be given or for a valid extension to be rejected, let's reverse it. 

Instead, we should try to refactor the update command to use the extract/verify extension functions used by the auto-pr workflow, since these provide a robust check against using an example file since #1034